docs/maintain-guides-how-to-monitor-your-node.md

```diff
@@ -0,0 +1,288 @@
+  --web.console.templates=/etc/prometheus/consoles \
+  --web.console.libraries=/etc/prometheus/console_libraries
+  ExecReload=/bin/kill -HUP $MAINPID
+
+[Install]
+  WantedBy=multi-user.target
+```
+
+Once the file is saved, execute the command below to reload `systemd` and enable the service so that
+it will be loaded automatically during the operating system's startup.
+
+```bash
+sudo systemctl daemon-reload && systemctl enable prometheus && systemctl start prometheus
+```
+
+Prometheus should be running now, and you should be able to access its front again end by
+re-visiting `IP_ADDRESS:9090/`.
+
+## Installing Grafana
+
+In order to visualize your node metrics, you can use Grafana to query the Prometheus server. Run the
+following commands to install it first.
+
+```bash
+sudo apt-get install -y adduser libfontconfig1
+wget https://dl.grafana.com/oss/release/grafana_7.1.3_amd64.deb
+sudo dpkg -i grafana_7.1.3_amd64.deb
+```
+
+If everything is fine, start the Grafana server and access it by going to the
+`http://SERVER_IP_ADDRESS:3000/login`. The default user and password is admin/admin.
+
+Now configure Grafana to auto-start on boot.
+
+> Note: If you want to change the port on which Grafana runs (3000 is a popular port), edit the file
+> `/usr/share/grafana/conf/defaults.ini` with a command like
+> `sudo vim /usr/share/grafana/conf/defaults.ini` and change the `http_port` value to something
+> else. Then restart grafana with `sudo systemctl restart grafana-server`.
+
+```bash
+sudo systemctl daemon-reload && sudo systemctl enable grafana-server && sudo systemctl start grafana-server
+```
+
+![grafana-1](assets/guides/how-to-monitor/1-grafana-login.png)
+
+In order to visualize the node metrics, click _settings_ to configure the `Data Sources` first.
+
+![grafana-1](assets/guides/how-to-monitor/2-add-data-source.png)
+
+Click `Add data source` to choose where the data is coming from.
+
+![grafana-1](assets/guides/how-to-monitor/2-add-data-source-2.png)
+
+Select `Prometheus`.
+
+![grafana-1](assets/guides/how-to-monitor/3-select-prometheus.png)
+
+The only thing you need to input is the `URL` that is `https://localhost:9090` and then click
+`Save & Test`. If you see `Data source is working`, your connection is configured correctly.
+
+![grafana-1](assets/guides/how-to-monitor/4-configure-data-source.png)
+
+Next, import the dashboard that lets you visualize your node data. Go to the menu bar on the left
+and mouse hover "+" then select `Import`.
+
+`Import via grafana.com` - It allows you to use a dashboard that someone else has created and made
+public. You can check what other dashboards are available via
+[https://grafana.com/grafana/dashboards](https://grafana.com/grafana/dashboards). In this guide, we
+use ["My Polkadot Metrics"](https://grafana.com/grafana/dashboards/12425), so input "12425" under
+the id field and click `Load`.
+
+![grafana-1](assets/guides/how-to-monitor/5-import-dashboard.png)
```

Comment:
is it possible to make these vertical images a bit smaller? They're quite large IMO